### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by lamblin**

### CODE REVIEW

**Comment by lamblin** [comment]
Second commit fixes #1155

**Comment by lamblin** [comment]
Third commit fixes #1385

File: pylearn2/scripts/tutorials/convolutional_network/tests/test_convnet.py [hunk 1]
```diff
@@ -14,16 +14,19 @@
...
-
+    """Test smaller version of convolutional_network.ipynb"""
     skip.skip_if_no_data()
     yaml_file_path = os.path.abspath(os.path.join(os.path.dirname(__file__),
                                                   '..'))
     save_path = os.path.dirname(os.path.realpath(__file__))
+    # Escape potential backslashes in Windows filenames, since
+    # they will be processed when the YAML parser will read it
+    # as a string
+    save_path.replace('\\', r'\\')
```
Comment (memimo) [inline pylearn2/scripts/tutorials/convolutional_network/tests/test_convnet.py]: If this does not break in unix systesm, then PR sounds good to me.

Comment (lamblin) [inline pylearn2/scripts/tutorials/convolutional_network/tests/test_convnet.py]: This works on unix too, and may even fix cases where there is a litteral backslash in a path.

**Pull request merged**